src/main/webapp/js/studentProfile.js

```diff
@@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
         	$('#statusMessage').css("display", "block")
         	        .attr('class', 'alert alert-danger')
         	        .html('There seems to be a network error, please try again later');
-        	$("html, body").animate({
-        		scrollTop: 0
-        	});
+        	scrollToTop({duration: ''});
```

Comment:
jQ takes strings for its animation methods. Any string aside from "fast" or
"slow" is interpreted as "normal"or default.

I replicated the behaviour by equating no parameter to no animation and
with parameter to animation according to the value.

I suppose a comment for that method would be a good idea, though.
On 16 Sep 2015 12:19, "Yap Jun Hao" <notifications@github.com> wrote:

> In src/main/webapp/js/studentProfile.js
> <https://github.com/TEAMMATES/repo/pull/4293#discussion_r39592624>:
>
> > @@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
> >          	$('#statusMessage').css("display", "block")
> >          	        .attr('class', 'alert alert-danger')
> >          	        .html('There seems to be a network error, please try again later');
> > -        	$("html, body").animate({
> > -        		scrollTop: 0
> > -        	});
> > +        	scrollToTop({duration: ''});
>
> This seems different from one of the other calls where no arguments is
> passed in, perhaps we should standardize?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TEAMMATES/repo/pull/4293/files#r39592624>.
>